spectacles/validators/data_test.py

```diff
@@ -50,6 +50,16 @@ def validate(self) -> Dict[str, Any]:
             model_name = test["model_name"]
             explore_name = test["explore_name"]
             query_url_params = test["query_url_params"]
+            file_path = test["file"].split("/", 1)[1]
+            explore_url = (
+                f"{self.client.base_url}/explore/{model_name}"
+                f"/{explore_name}?{query_url_params}"
+            )
```

Comment:
I guess the earlier PR didn't actually implement this?

Reply:
It did. I'm just moving it outside of the loop below as we already have all the info and need it for both the error and success path so it stops us from declaring it twice.